src/engine/renderer/glsl_source/computeLight_fp.glsl

```diff
@@ -75,9 +75,21 @@ void ReadLightGrid(in vec4 texel, out vec3 ambientColor, out vec3 lightColor) {
 }
 #endif
 
-#if defined(USE_DELUXE_MAPPING) || defined(USE_GRID_DELUXE_MAPPING) || (defined(r_realtimeLighting) && r_realtimeLightingRenderer == 1)
+#if defined(USE_DELUXE_MAPPING) || defined(USE_GRID_DELUXE_MAPPING) || defined(r_realtimeLighting)
+#if !defined(USE_PHYSICAL_MAPPING)
+#if defined(r_specularMapping)
 uniform vec2 u_SpecularExponent;
```

Comment:
The fix is that this `uniform vec2 u_SpecularExponent;` line is now used for both the forward and the tiled light renderer.